RadixWallet/Features/SettingsFeature/SettingsRow.swift

```diff
@@ -0,0 +1,115 @@
+import SwiftUI
+
+// MARK: - SettingsRow
+enum SettingsRow<Feature: FeatureReducer> {
+	/// A standard tappable row with the details specified on the `Model`
+	case model(Model)
+
+	/// A custom row with its own UI. Useful, for example, when we want a `ToggleView`.
+	case custom(AnyView, id: String)
```

Comment:
I suppose the AnyView is unavoidable if we want a completely custom row type. But do we actually need that? Seems to go somewhat against the core idea, will we really have a bunch of completely custom rows in settings?

Reply:
not right now at least. We could delete the entire concept and in the usage of it, instead of simply iteration over an array of `[SettingsRow]`, we would iterate over a first array of `[SettingsRow]`, then add the custom one, then iterate over an array of remaining `[SettingsRow]`. 

I can remove it for now and if in the future we have more custom rows we consider adding it back.

Reply:
Is this added to be able to have a `toggle` in a settings row?

Reply:
Yes @GhenadieVP, will remove it though.

Reply:
ok, @matiasbzurovski I am thinking that it could be worth extending PlainListRow to accept arbitrary accessory view, unless you are doing exactly this alread.

Reply:
not really, I am reusing the `ToggleView` which internally uses `PlainListRowCore`